cmd/hz/protobuf/tags.go

```diff
@@ -270,10 +270,10 @@ func reflectJsonTag(f protoreflect.FieldDescriptor) (ret model.Tag) {
 	if v := checkFirstOption(api.E_Body, f.Options()); v != nil {
 		ret.Value += ",string"
 	}
-	if !unsetOmitempty && descriptorpb.FieldDescriptorProto_Label(f.Cardinality()) == descriptorpb.FieldDescriptorProto_LABEL_OPTIONAL {
-		ret.Value += ",omitempty"
-	} else if descriptorpb.FieldDescriptorProto_Label(f.Cardinality()) == descriptorpb.FieldDescriptorProto_LABEL_REQUIRED {
```

Comment:
Why stop checking this`optional` tag?

Reply:
[protobuf-go](https://github.com/protocolbuffers/protobuf-go/blob/bc1253ad37431ee26876db47cd8207cdec81993c/cmd/protoc-gen-go/internal_gengo/main.go#L722) 

1.  protoc-gen-go will add "omitempty" to every field by default. 
2. Some adjustments were made to this behavior, adding "required option" to "required filed" and "omitempty option" to "optional field". Usually the label for "default field" is also "optional"
3. However, "repeated field" & "map field" are not "optional field" by default.